json_to_csv.py

```diff
@@ -0,0 +1,46 @@
+import os
+import json
+import csv
+
+
+def fun(in_dir,out_dir,**d):
+
+    os.chdir("C:/Users/anurag/Downloads/darkflow-master/darkflow-master") #change directory to that path where object detector YOLO v2 resides.
+    if not os.path.exists(in_dir):
+        print("INVALID DIRECTORY ENTERED")
+    else:
+        os.system("python flow --imgdir "+in_dir+"/ --model cfg/yolo.cfg --load bin/yolo.weights --json")
+
+        filelst = os.listdir(in_dir+"/out")

```

Comment:
why this concatenation? in_dir + `out`?

Reply:
Output json files  will be always be stored  in the automatically created 'out' folder inside the 'in_dir' path. Here I am storing all the names of the json files in the list named 'filelst' for further use in creating the corresponding csv file with the same name.